content/en/docs/started/architecture.md

```diff
@@ -4,142 +4,147 @@ description = "An overview of Kubeflow's architecture"
 weight = 10
 +++
 
-<!--
-Note for authors: The source of the diagrams is held in Google Slides decks,
-in the "Doc diagrams" folder in the public Kubeflow shared drive.
--->
-
-This guide introduces Kubeflow as a platform for developing and deploying a
-machine learning (ML) system.
-
-Kubeflow is a platform for data scientists who want to build and experiment with
-ML pipelines. Kubeflow is also for ML engineers and operational teams who want
-to deploy ML systems to various environments for development, testing, and
-production-level serving.
+This guide introduces Kubeflow ecosystem and explains how Kubeflow components fit in ML lifecycle.
 
-## Conceptual overview
+Read [the introduction guide](/docs/started/introduction) to learn more about Kubeflow, standalone
+Kubeflow components and Kubeflow Platform.
 
-Kubeflow is *the ML toolkit for Kubernetes*.
+## Kubeflow Ecosystem
 
-The following diagram shows Kubeflow as a platform for arranging the
+The following diagram shows Kubeflow ecosystem for arranging the
 components of your ML system on top of Kubernetes:
 
 <img src="/docs/started/images/kubeflow-architecture.drawio.svg"
   alt="An architectural overview of Kubeflow on Kubernetes"
-  class="mt-3 mb-3 border border-info rounded">
+  class="mt-3 mb-3">
 
-Kubeflow builds on [Kubernetes](https://kubernetes.io/) as a system for 
+Kubeflow builds on [Kubernetes](https://kubernetes.io/) as a system for
 deploying, scaling, and managing complex systems.
 
-Using the Kubeflow configuration interfaces (see [below](#interfaces)) you can
-specify the ML tools required for your workflow. Then you can deploy the 
-workflow to various clouds, local, and on-premises platforms for experimentation and 
-for production use.
+Using the Kubeflow configuration interfaces (see [below](#kubeflow-interfaces)) you can
+specify the ML tools required for your use cases.
```

Comment:
I am not sure we need these lines under here, as they aren't about the ecosystem, and I don't like using the word "complex".

Also, it's not really correct to say the interfaces let you choose which ML tools to use, that's the deployment process.

Reply:
I will say this:
```
Kubeflow builds on [Kubernetes](https://kubernetes.io/) as a system for
deploying, scaling, and managing AI/ML infrastructure.
```

Reply:
>Also, it's not really correct to say the interfaces let you choose which ML tools to use, that's the deployment process.

Let me remove this sentence for now.